client/components/follow-button/button.jsx

```diff
@@ -11,7 +12,8 @@ const FollowButton = React.createClass( {
 		onFollowToggle: PropTypes.func,
 		iconSize: PropTypes.number,
 		tagName: PropTypes.oneOfType( [ PropTypes.string, PropTypes.func ] ),
-		disabled: PropTypes.bool
+		disabled: PropTypes.bool,
+		followLabel: PropTypes.string
```

Comment:
Yep - so we could potentially add `followingLabel` in the future if it was needed.